molecule/command/init/scenario.py

```diff
@@ -82,7 +82,9 @@ def execute(self):
             ).format(scenario_name)
             util.sysexit_with_message(msg)
 
-        driver_template = 'scenario/driver/{driver_name}'.format(**self._command_args)
+        driver_template = api.drivers()[
+            self._command_args['driver_name']
```

Comment:
Uhhhh, this is always taking the driver from the CLI invocation and not from a configuration like `molecule.yml`?

Reply:
They are not affected, there is a list of template directories, all loaded.

PS. Yes, I am really ware that the construct is very ugly but I did not want to improve code while fixing bugs. We can work to make the code more pythonic later.